sklearn/svm/classes.py

```diff
@@ -594,6 +602,10 @@ def __init__(self, C=1.0, kernel='rbf', degree=3, gamma='auto_deprecated',
             decision_function_shape=decision_function_shape,
             random_state=random_state)
 
+    @classmethod
+    def _get_test_instances(cls):
+        yield cls(decision_function_shape='ovo')
```

Comment:
This will fail, right?

Reply:
Or rather: why doesn't this fail?

Reply:
The current `set_checking_parameters` does this for *SVC

Reply:
you're right. But why? I'm sure that was me. I should get some coffee and think about that. That makes the shape of the decision function wrong, doesn't it? Are we never testing for more than 3 classes? I'm quite confused.